LUNAR_LANDING_GUIDANCE_EQUATIONS.s

```diff
@@ -415,7 +415,7 @@ REDES1		DLOAD	DSU
 #	VELOCITY RELATIVE TO THE SURFACE
```

Comment:
Missing colon after `SURFACE` - should be `SURFACE:`
[Source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0805.jpg)